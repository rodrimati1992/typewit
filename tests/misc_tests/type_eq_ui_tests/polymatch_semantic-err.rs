struct Foo;

fn main(){
    typewit::polymatch! {100;
        200 | 300 => {
            impl Foo {
                fn foo() {}
            }
        }
    };
}