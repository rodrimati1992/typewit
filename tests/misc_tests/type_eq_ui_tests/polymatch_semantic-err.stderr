error[E0592]: duplicate definitions with name `foo`
 --> tests/misc_tests/type_eq_ui_tests/polymatch_semantic-err.rs:7:17
  |
7 |                 fn foo() {}
  |                 ^^^^^^^^
  |                 |
  |                 duplicate definitions for `foo`
  |                 other definition for `foo`
